msal/src/main/java/com/microsoft/identity/client/controllers/MSALInteractiveTokenCommand.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.identity.client.controllers;
+
+import android.content.Intent;
+
+import com.microsoft.identity.client.AuthenticationResult;
+
+import java.util.concurrent.ExecutionException;
+
+public class MSALInteractiveTokenCommand extends MSALTokenCommand {
+
+    @Override
+    public AuthenticationResult execute(){
+        if(getParameters() instanceof MSALAcquireTokenOperationParameters){
+            AuthenticationResult result = null;
+
+            try {
+                result = getController().AcquireToken((MSALAcquireTokenOperationParameters) getParameters());
+            } catch (ExecutionException e) {
```

Comment:
since we are doing the same operation in both catch blocks we can use the syntax ExecutionException | InterruptedException

Reply:
Sure... i'm letting the IDE generate some code.  If there's a styleguide that we should be following let me know.

Reply:
I'm going to leave this as is for now.... so that I remember to actually implement this properly.  I've added a couple of TODOs.